src/release.py

```diff
@@ -344,6 +345,7 @@ def run():
         resp = fetch_history_from_github(github_user, github_project, first_commit)
 
     commits_since_tag = []
+    bug_name_prefix_regexp = re.compile("bug ([\d]+)", re.IGNORECASE)
```

Comment:
Since this has a `\d` and we want that to mean "regexp digit" and not "escaped d in the string", it should be a raw string and needs the `r` at the beginning.

Also, this is backwards--we want the regexp to match `bug-nnnnnnn` so we can convert that to `bug nnnnnnn` when running `make-bug` so that bugzilla can do its autolinking.

It's a little confusing since `release.py` has two subcommands and we only want the replacing to happen for the `make-bug` command.